packages/netlify-cms/package.json

```diff
@@ -22,6 +22,7 @@
   "license": "MIT",
   "dependencies": {
     "codemirror": "^5.46.0",
+    "create-react-class": "^15.7.0",
```

Comment:
This was a missing dependency. The reason it worked until now is because it was a dependency of `react-datetime`